kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CustomResourceCrud1109Test.java

```diff
@@ -60,7 +62,11 @@ void testRequestsArePerformedToDashedPath() throws Exception {
     // When
     fooBarClient.inNamespace("my-namespace").list();
     // Then
-    assertThat(server.getLastRequest())
+
+    int requestCount = server.getRequestCount();
+    RecordedRequest lastRequest = null;
+    while(requestCount-- > 0)lastRequest = server.takeRequest();
```

Comment:
Can we move this last request calculation logic to some util method?

Reply:
Done